src/components/Modal/Strategy.vue

```diff
@@ -87,23 +110,51 @@ watch(open, () => {
     <div v-if="input.name" class="m-4">
       <RowLoading v-if="loading" class="px-0" />
       <div v-else>
-        <SDefaultObject
-          v-if="strategyDefinition"
-          v-model="input.params"
-          :definition="strategyDefinition"
-        />
-        <UiButton
-          v-else
-          class="block w-full mb-3 overflow-x-auto"
-          style="height: auto"
-        >
-          <TextareaJson
+        <div class="min-h-[280px]">
+          <BaseAutocomplete
+            :options="networks"
+            v-model="input.network"
+            title="Network"
+            class="mb-3"
+          >
+            <template v-slot:option="{ option }">
+              <div class="group flex items-center justify-between">
+                <div class="flex items-center">
+                  <UiAvatar
```

Comment:
This network image could be a little smaller maybe 
<img width="397" alt="image" src="https://user-images.githubusercontent.com/51686767/155287158-88c153e7-1a98-422c-acdd-b2e1e3406678.png">